app/controllers/concerns/pageable_feed_concern.rb

```diff
@@ -0,0 +1,14 @@
+module PageableFeedConcern
+  extend ActiveSupport::Concern
+  include PageableConcern
+
+  private
+
+  def initialize_pageable_feed
+    page = params[:page]&.to_i || 1
+
+    feed_items = Feed.for_page(:title, page, items_per_page)
```

Comment:
NOTE: Performance. Use of the Feed.for_page scope that limits our database hits to only what we need per page, rather than pulling the whole table content with each postback/ajax call.